### PR TITLE
Migrate pachakges

### PR DESCRIPTION
**Pull request by MatthysDucrocq**

### CODE REVIEW

File: src/components/quote/quoteItem.js [hunk 1]
```diff
@@ -1,71 +1,32 @@
 import React from 'react'
```
Comment (flexbox) [inline src/components/quote/quoteItem.js:1]: Can you migrate this to 
`components/QuoteItem.js` 🙏

File: src/components/quote/quoteItem.js [hunk 2]
```diff
@@ -1,71 +1,32 @@
...
-    <Section className="max-w-screen-md rounded bg-white shadow transition duration-500 ease-in-out transform hover:shadow hover:scale-4">
-      <Blockquote className="text-center m-4 p-4">
-        <Title className="font-vidaloka font-semibold pt-12 pb-6 text-5xl">
+    <div
+      id="Section"
+      className="max-w-screen-md rounded bg-white shadow transition duration-500 ease-in-out transform hover:shadow hover:scale-4"
+    >
+      <div id="blockquote" className="text-center m-4 p-4">
+        <div
+          id="Title"
```
Comment (flexbox) [inline src/components/quote/quoteItem.js:11]:
```suggestion
          id="title"
```

File: src/components/share/share.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import React from 'react'
```
Comment (flexbox) [inline src/components/share/share.js:1]: `Share.js` 🙏

File: src/pages/404.js [hunk 4]
```diff
@@ -1,26 +1,20 @@
 import React from 'react'
 import { Link } from 'gatsby'
 import Layout from 'components/layout'
-import styled from 'styled-components'
-import { Heading } from 'rebass/styled-components'
 
 const NotFoundPage = () => (
   <Layout>
-    <Heading as="h1">NOT FOUND</Heading>
+    <div id="Heading" as="h1">
```
Comment (flexbox) [inline src/pages/404.js:7]: `as` is for rebass 
here it's better if you write `<h1>...</h1>`

**Review by flexbox — CHANGES_REQUESTED** [review]
I don't think you need to add id's 
Example:
```
id="flex" 
```
Seams useless, please remove all `id=`

**Comment by MatthysDucrocq** [comment]
Fix

**Pull request merged**